frontend/src/App.svelte

```diff
@@ -26,11 +27,13 @@
   let selected_node = null;
 
   let show_menu = false;
-  let show_legend = false;
+  let show_legend = true;
   let show_timebar = false;
```

Comment:
Missing type declarations

Reply:
This part is not purely my part. Should I still add the types?

Reply:
I would say yes, we need to adhere to the typescript standard and make sure that things throw errors if they are changed in a way that is not expected.